frappe/public/js/frappe/form/toolbar.js

```diff
@@ -378,7 +378,8 @@ frappe.ui.form.Toolbar = class Toolbar {
 				function () {
 					me.frm.copy_doc();
 				},
-				true
+				true,
+				"Ctrl+D"
```

Comment:
Ctrl+D is a standard shortcut in browsers for bookmarking things. We shouldn't override it.